todo/input.py

```diff
@@ -209,17 +209,19 @@ def smart_menu(card, f_display, list_choices, label_choices, color=None):
     @staticmethod
     def title_to_link(card):
         # assumes card.name is the link you want
-        links = [n for n in card.name.split() if 'http' in n]
+        sp = card.name.split()
+        links = [n for n in sp if 'http' in n]
```

Comment:
It'd be slower, but also more robust to check against a URL regex, since `http://` could be omitted.

Reply:
Performance on this isn't too big a concern for me so I'm going to add a regular expression. I don't want to exclude urls that aren't perfectly formed.

Reply:
I can't find a good balance of regex that isn't overly broad, catching arbitrary words with periods in them. I'm going to apply a regex evenly across the codebase that mandates the use of `https?://.*\.` in the link.